src/ripple/app/ledger/Ledger.cpp

```diff
@@ -527,7 +527,7 @@ Ledger::rawInsert(std::shared_ptr<SLE> const& sle)
     auto item = std::make_shared<
         SHAMapItem const>(sle->key(),
             std::move(ss));
```

Comment:
So `Serializer` doesn't have a move constructor and this is a prime use case for having one. @HowardHinnant, should we add one to it (along with a move assignment, I guess), even if it's just `= default`.

Reply:
@nbougalis I believe the compiler will supply a move constructor for `Serializer`